src/member/models.py

```diff
@@ -151,6 +151,9 @@ def add_contact_information(self, type, value, force_update=False):
         updated. Otherwise, it should create a new one.
         """
         created = False
+        if value is None:
+            value = ''
+
         if force_update or value is not '':
             contact, created = Contact.objects.update_or_create(
```

Comment:
@fpeyre, question here: Why not use `if force_update or value is not None:` instead of adding the if condition?